_episodes/running-containers.md

```diff
@@ -264,18 +267,18 @@ just type `exit`.
 >
 > > ## Solution 2 -- Run commands
 > >
-> > Run the busybox container, first with a command to read out the Linux version:
+> > Run a busybox container, first with a command to read out the Linux version:
 > > ~~~
 > > $ docker container run busybox cat /proc/version
 > > ~~~
 > > {: .language-bash}
 > >
-> > Then run the container again with a command to print out the busybox help:
+> > Then run a container with a command to print out the busybox help:
 > > ~~~
 > > $ docker container run busybox busybox --help
 > > ~~~
 > > {: .language-bash}
-> > Note that this command uses `busybox` twice, first for the image name and then to run the `busybox` command within the busybox container that is started.
+> > Note that this command runs a `busybox` container twice, first to retrieve the version of Linux from the running container and then to run the `busybox` command within the busybox container that is started.
```

Comment:
I find this change a bit confusing.  I don't think it runs the container twice. 
The first `busybox` isn't to retrieve the version of linux from the running container, it is to start up a container using the busybox container image.  Then the 2nd one calls the `busybox` command within the new busybox container that is running.

Reply:
@sstevens2 I am confused here I think. When I run the first command:

```
docker container run busybox cat /proc/version
```

My understanding is that this starts the container from the `busybox:latest` container image (pulling if necessary) and then runs the command `cat /proc/version`. The container then exits (I can see in the Docker console that the image is exited). When you then run the second command:

```
docker container run busybox busybox --help
```

Docker, again, starts the container, this time runs the command `busybox --help` and then exits the container again. This is why I think the container has been run twice. The fact that the container is "exited" rather than removed does not matter here, this is just a convenience from Docker to improve response speed (i.e. I could add the `--rm` option in both cases and it would still work in the same way). The container does not keep running in between the two commands and, if you made changes, the state would not be saved (unless you committed them to a new container image with a different name from the image you are using).

Reply:
Ahh I see what you are thinking now.  I think we are differing in how we interpret the note.  I think the note is specifically about the 2nd command, trying to explain why the word `busybox` is repeated in the 2nd command (once for the container and once for the command inside the containers).  
Your change makes the note about running the busybox container twice in two separate commands executed instead. Maybe we need to clarify the language around this to indicate that both that the busybox container is run twice with two different executed commands and that in the 2nd command `busybox` is repeated twice, once for the container and once for the executed command.  

Let me know if this difference is still unclear. I had a hard time trying to explaining the difference in interpretation.